provider/cldr/src/transform/time_zones/convert.rs

```diff
@@ -46,14 +45,15 @@ impl From<TimeZoneNames> for TimeZoneFormatsV1<'_> {
                 .into_iter()
                 .map(|(key, value)| {
                     (
-                        Cow::Owned(
-                            key.parse::<TinyStr8>()
-                                .expect("Time-zone variant was not compatible with TinyStr8"),
-                        ),
-                        value.into(),
+                        key.parse::<TinyStr8>()
+                            .expect("Time-zone variant was not compatible with TinyStr8"),
+                        value,
                     )
                 })
-                .collect(),
+                .fold(ZeroMap::new(), |mut map, (key, value)| {
+                    map.insert(&key, &value);
+                    map
+                }),
```

Comment:
Suggestion (optional), here and below: It would be nicer if you implemented FromIterator on ZeroMap.